projects/payment-lib/src/lib/components/payment-view/payment-view.component.ts

```diff
@@ -413,11 +413,22 @@ export class PaymentViewComponent implements OnInit {
 
   chkIsAddRemissionBtnEnable(fee: IFee): boolean {
     if (fee !== null && fee !== undefined) {
-      return fee.add_remission && fee.remission_enable;
+      return fee.add_remission && fee.remission_enable && this.isTheCurrentRefundInProcessForThisFee(fee);
     } else {
       return false
     }
   }
+
+  // This method is going to check if the current refund has been rejected.
+  // If this is the case the button should be disable.
+  isTheCurrentRefundInProcessForThisFee(fee: IFee): boolean{
```

Comment:
You have the same method defined twice in payment-vew.component.ts and service-request.component.ts? 

It's using the paymentLibComponent, so this needs to be added to the payment-lib.component.ts as you've duplicated the same code. 

You can then call :
paymentLibComponent.isTheCurrentRefundInProcessForThisFee(fee);